docs/attributes.md

```diff
@@ -38,6 +38,8 @@ Configurable (with defaults)
 * `node['splunk']['config']['license-pool']` - Data bag item used to configure license pools.
 * `node['splunk']['config']['ui_prefs']` - Hash of stanzas used to configure [ui-prefs.conf][] on the search head in a clustered configuration or a standalone instance.
 * `node['splunk']['config']['assumed_index']` - Name of the index to which data is forwarded to by default, when the index is not configured for the input.(`main`)
+* `node['splunk']['config']['sh_cluster']` - Set this attribute to true to enable SH clustering. (`false`)
```

Comment:
We should probably spell out Search Head the first time and enclose SH in parenthesis so that it sets the precedent which can be used throughout this document like what you did for the databags markdown file.